INTEGRATION.rst

```diff
@@ -0,0 +1,156 @@
+
+Continuous Integration
+======================
+
```

Comment:
+1, mention something shorter about testing in readme and more important describe testing in documentation in salt-formulas repo.
We don't want to duplicate the same long text in each repo.